bigbluebutton-html5/imports/api/users/server/methods/userLeaving.js

```diff
@@ -30,11 +30,11 @@ export default function userLeaving(credentials, userId) {
       'user-not-found', `You need a valid user to be able to toggle audio`);
   }
 
-  if (User.user.connection_status === OFFLINE_CONNECTION_STATUS) {
+  if (User.connection_status === OFFLINE_CONNECTION_STATUS) {
```

Comment:
I checked the object we receive here - this is what's in the parent 'User' object:
![user](https://cloud.githubusercontent.com/assets/9299997/23873884/04a17c92-080a-11e7-9b01-c472abd70a10.JPG)
It looks like there is no nested user object there..